claude_scripts/confidence_tracker.py

```
@@ -0,0 +1,287 @@
+            return "positive"
+        return "neutral"
+
+    def update_pattern_confidence(self, pattern_ids, user_feedback):
+        """Update confidence for applied patterns based on user feedback"""
+        if not MEMORY_FILE.exists():
+            return "No memory file found"
+
+        try:
+            with open(MEMORY_FILE) as f:
+                memory = json.load(f)
+        except:
```

Ruff (0.14.4)

Do not use bare `except`

(E722)